.github/workflows/ci.yml

```
@@ -0,0 +1,141 @@
+            tag: dbr-echo-directus
+            build_args: ""
+          # Shared image for server and worker
+          - name: dbr-echo-server
+            context: ./echo/server
+            dockerfile: Dockerfile
+            tag: dbr-echo-server
+            build_args: ""
+    
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)